Assets/Scripts/TapeInteract.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.InputSystem;
+
+/// <summary>
+/// This script should be added on the PlayerControler Object.
+/// Usess raycasts to see whats in front of the player, to be interacted with.
+/// Shows tooltip if Player can interact with object
+/// 
+/// Requires: The tape object tag to be set to "Tape"
+/// </summary>
+
+
+public class TapeInteract : MonoBehaviour {
+    [Tooltip("Add the playerObject here")]
+    [SerializeField]
+    GameObject PlayerObject;
+
+    [Tooltip("Set the distance of the raycast for interacting with tapes(Should be arms lenght)")]
+    [SerializeField]
+    float distanceToInteract;
+
+    [SerializeField]
+    GameObject pressEToolTip;
+
+    float playerReach = 1;
+
+    RaycastHit hit;
+    Vector3 playerForwardDirection;
+
+    // Start is called before the first frame update
+    void Start() 
+    {
+        if (pressEToolTip == null) 
+        {
+            //Look up Canvas, and then the pressE Text/Game Object 
+            pressEToolTip = GameObject.Find("Canvas").transform.Find("PressE").gameObject;
+
+        }
+
+        pressEToolTip.SetActive(false);
+    }
+
+    // Update is called once per frame
+    void Update() 
+    {
+        ShootRaycast();
+
+    }
+
+    void ShootRaycast() 
+    {
+
+        //get player forward direction
+        playerForwardDirection = PlayerObject.transform.TransformDirection(Vector3.forward);
```

Comment:
`transform.forward` should be the same thing. Try to test it. And code convention.

```suggestion
        playerForwardDirection = playerObject.transform.forward;
```